source/isaaclab/isaaclab/envs/mdp/events.py

```diff
@@ -1188,8 +1188,25 @@ def __init__(self, cfg: EventTermCfg, env: ManagerBasedEnv):
             body_names_regex = ".*"
 
         # create the affected prim path
-        # TODO: Remove the hard-coded "/visuals" part.
-        prim_path = f"{asset.cfg.prim_path}/{body_names_regex}/visuals"
+        # Check if the pattern with '/visuals' yields results when matching `body_names_regex`.
```

Comment:
Can you please modify the unit test for this function to showcase the cases that's getting handled here?

Reply:
Hi @Mayankm96, Thank you for the suggestion. The change I'm implementing aims to handle cases where the visual prim path does not end with visuals. I've checked the assets used in the test cases, and all of them end with visuals.

In my use case, the table prim path I'm working with is something like /World/envs/env_.*/Table/.*. I haven't found any assets that could show this benefit in the test file. Do you have any suggestions?

Thank you!

Reply:
@Mayankm96, Could you please take a look at the test I added? I mocked the `find_matching_prim_paths` function to return an empty list, allowing it to trigger the fallback behavior. This tests the scenario where the /visuals pattern doesn't match. thanks.